physionet-django/templates/footer.html

```diff
@@ -1,18 +1,60 @@
-<!-- This footer template is from the bootstrap 'blog' example -->
 {% load static %}
 <link rel="stylesheet" type="text/css" href="{% static 'custom/css/footer.css' %}"/>
 
-<footer class="blog-footer">
-  <div class="container">
-    <div class="row row-centered">
-      <div class="col-xs-10 center col-centered">
-        <p>{{ SITE_NAME }} is a repository of freely-available medical research data, managed by the {{ FOOTER_MANAGED_BY }}.</p>
-        <p>Supported by the {{ FOOTER_SUPPORTED_BY }}.</p>
-        {% if FOOTER_ACCESSIBILITY_PAGE %}
-          <p>For more accessibility options, see {{ FOOTER_ACCESSIBILITY_PAGE|safe }}.</p>
-        {% endif %}
-        <p><a href="#">Back to top</a></p>
+<div class="modern-ui">
+  <footer class="footer w-100 secondary-bg py-5">
+    <div class="container my-0 py-0 py-md-5">
+      <img src="{% static 'images/logo-white.png' %}" width="180px" alt="PhysioNet Logo" class="pb-2" />
+      <div class="footer__columns light-text">
+        <div>
+          <p class="dark-light-text mb-2">PhysioNet is a repository of freely-available medical research data, managed by the MIT Laboratory for Computational Physiology.</p>
+          <p class="dark-light-text mb-2">Supported by the National Institute of Biomedical Imaging and Bioengineering (NIBIB) under NIH grant number R01EB030362.</p>
+          <p class="dark-light-text mb-2">For more accessibility options, see the <a class="dark-primary-text" href="https://accessibility.mit.edu/">MIT Accessibility Page.</a></p>
+        </div>
+        <div class="d-flex flex-column pl-0 pl-md-5 pt-4 pt-md-0">
+          <h2 class="text-white font-weight-bold">Navigation</h2>
+          <a href="/content" class="pb-3 dark-light-text">Discover Data</a>
```

Comment:
These menu items ("about", "share data" etc) should be pulled in from the static pages model, not hardcoded. One the static page management page (e.g. https://physionet.org/console/static-page/2/edit/) there is a item called "nav bar" that determines whether the page appears in the nav bar.